server/server.js

```diff
@@ -25,6 +26,7 @@ app.use(passport.session());
 /* Routes */
 app.use('/api/user', userRouter);
 app.use('/api/messages', messagesRouter);
+app.use('/api/', surveyRouter);
```

Comment:
does this need to be /api/survey?

Reply:
I have a '/' for the user in the survey.router.js so I have the  '/survey' endpoint in the router instead...

Reply:
understood, but the rest of the endpoints do it the way I suggested. Your way functions just fine, but you are just doing it differently and it adds some confusion to the code, especially because we are going to hand this code off in a few weeks for the client to try to understand